koji/foreman-client-mash-split.py

```diff
@@ -318,9 +318,8 @@ def main():
     else:
         git_tag = "rpm/{}".format(version)
 
-        if version in ('1.20', '1.21'):
-            del dists['fedora29']
-            dists['fedora27'] = 'fc27'
+        if version in ('1.22', '1.23', '1.24'):
```

Comment:
yes, this explicitly lists all current versions, that way we can just drop that when we only have el8-supported released